caravel/assets/javascripts/modules/caravel.js

```diff
@@ -76,16 +77,13 @@ const px = function () {
       container,
       containerId,
       selector,
-      querystring(params) {
-        const newParams = params || {};
+      querystring() {
         const parser = document.createElement('a');
         parser.href = data.json_endpoint;
-        if (dashboard !== undefined) {
-          let flts = '';
-          if (newParams.extraFilters !== false) {
-            flts = dashboard.effectiveExtraFilters(sliceId);
-            flts = encodeURIComponent(JSON.stringify(flts));
-          }
+        if (controller.type === 'dashboard') {
```

Comment:
thanks for the review, I know there's much more refactoring to do here to move all the logic in the right places, I want to merge this quickly-ish to fix the bug and we can see how much more refactoring we want to do here in the future but most likely not much knowing that we're re-writing the whole explore view...